ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/utils.py

```diff
@@ -281,7 +288,7 @@ def _get_l2_as_subint(self, link: dict, svi: dict, vrf: dict) -> dict:
 
         # TODO: enable flow tracking once toggle is in place
         # Configuring flow tracking on LAN interfaces of WAN routers
-        # if self.shared_utils.is_cv_pathfinder_client:
-        #    subinterface["flow_tracker"] = {"hardware": self.shared_utils.wan_flow_tracker_name}
+        if self.shared_utils.is_cv_pathfinder_client:
```

Comment:
should there be another condition here? I think thats why Flow tracker is enabled on sub interfaces for uplinks.

Reply:
I am not sure, what should be the behaviour for subinterfaces, should I use the uplinks variable, or something else?